### PR TITLE
Change webpack to build a UMD library

### PR DESCRIPTION
**Pull request by PeachIceTea**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = spaces
+indent_size = 4
```
Comment (InDieTasten) [inline .editorconfig]: I like you for that one 👍

File: .gitignore [hunk 2]
```diff
@@ -20,4 +21,4 @@ node_modules
 .npm
 
 # Output of 'npm pack'
-*.tgz
\ No newline at end of file
+*.tgz
```
Comment (InDieTasten) [inline .gitignore]: I like you for that one as well :+1:

File: package.json [hunk 3]
```diff
@@ -2,15 +2,17 @@
   "name": "isodom.js",
   "version": "0.0.0",
   "private": true,
-  "dependencies": {
-    "webpack": ""
-  },
   "scripts": {
-    "dev": "webpack-dev-server --inline --hot --colors --progress",
-    "build": "webpack --progress --colors -p"
+    "dev": "webpack-dev-server --inline --colors --progress --config build/dev.js",
```
Comment (InDieTasten) [inline package.json]: What about `serve` or `watch`?

Comment (PeachIceTea) [inline package.json]: We can alias them if you want but I think dev is the most accurate.

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,3110 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (PeachIceTea) [inline yarn.lock]: I would recommend going with yarn if that is not an issue.

File: package.json (hunk 3 above)
Comment (InDieTasten) [inline package.json]: Then leave it like that.

File: yarn.lock (hunk 4 above)
Comment (InDieTasten) [inline yarn.lock]: But does it make sense to add this file to version control? It says it's autogenerated. Same goes for dist and build folders, but these are at least convenient to have to have a form of distribution via the repo

Comment (PeachIceTea) [inline yarn.lock]: What yarn does here is layout the exact dependencies of everything. The purpose of the file is to provide more accurate dependency installation than npm. The functionality of that depends on that everyone has the same yarn.lock.

**Pull request merged**